doc/nrf/gs_programming.rst

```diff
@@ -102,7 +102,7 @@ Complete the following steps to build |NCS| projects with SES after :ref:`instal
    .. important::
       If you are working with an nRF9160 DK, make sure to select the correct controller before you program the application to your development kit.
 
-      Put the **SW5** switch (marked debug/prog) in the **NRF91** position to program the main controller, or in the **NRF52** position to program the board controller.
+      Set the **SW10** switch (marked debug/prog) in the **NRF91** position to program the main controller, or in the **NRF52** position to program the board controller. In DK v0.9.0 and earlier, the switch is called **SW5**.
```

Comment:
```suggestion
      Set the **SW10** switch (marked debug/prog) in the **NRF91** position to program the main controller, or in the **NRF52** position to program the board controller. 
      In nRF9160 DK v0.9.0 and earlier, the switch is called **SW5**.
```

Reply:
nRF9160 DK, not just DK (also in the other occurrences)

Also, please add the line break - I can look at the compliance failure if it occurs. There is no reason for a failure, and no reason to violate our style guide. ;)